src/program.ts

```diff
@@ -1239,7 +1239,7 @@ export class Program extends DiagnosticEmitter {
             } else {
               this.error(
                 DiagnosticCode.Expected_0_arguments_but_got_1,
-                decorator.range, "1", numArgs.toString(0)
```

Comment:
fix potentially `RangeError` like `toString() radix argument must be between 2 and 36`